R/oauth-init.R

```diff
@@ -57,6 +57,8 @@ init_oauth1.0 <- function(endpoint, app, permission = NULL,
 #'   Otherwise, provide a URL to the user and prompt for a validation
 #'   code. Defaults to the of the \code{"httr_oob_default"} default,
 #'   or \code{TRUE} if \code{httpuv} is not installed.
+#' @param oob_value if provided, is used as a custom oob callback value.
```

Comment:
Can you please make this description a bit more precise? It's not a callback; it's passed as a redirect parameter.

Reply:
Done.